### PR TITLE
Add instructions on how to install via Homebrew on OSX

### PR DESCRIPTION
**Pull request by ciarant**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,11 @@ Sometimes this is just a more practical and quick way than doing things properly
     chmod +x miniserve-osx
     ./miniserve-osx
 
+Alternatively install view [Homebrew](https://brew.sh/).
```
Comment (svenstaro) [inline README.md]: Probably a typo. I suggest: `Alternatively install with`

Comment (ciarant) [reply README.md]: Apologies, you are correct.

Comment (svenstaro) [reply README.md]: Cool, I'll merge once you change it.

Comment (ciarant) [reply README.md]: Done!

**Comment by svenstaro** [comment]
Cool, thanks. :)

**Pull request merged**